hack/make.sh

```diff
@@ -157,6 +168,9 @@ function install_pouch
 	install_runc
 	# copy pouch daemon and pouch cli to PATH
 	echo "Install pouch."
+	if [[ $CAL_INTEGRATION_TEST_COVERAGE != "no" ]]; then
```

Comment:
I think we can use the "yes" next time here, because we don't want to handle the convert in mind 😄